src/main/webapp/components/sources-info/sources-info.js

```diff
@@ -47,12 +47,13 @@ const sourcesMessage = offlineCount => {
 
 export const SourcesInfo = props => {
   const sources = props.sources === undefined ? [] : props.sources
-  const offlineCount = sources.filter(source => !source.isAvailable).length
+  const error = props.error === undefined ? '' : props.error
```

Comment:
:pencil2: don't need to modify the error prop
```suggestion
    const { error } = props
```